greenlight/tests/functional/keyboard_shortcuts/test_browser_window.py

```diff
@@ -39,9 +39,11 @@ def test_search_field(self):
 
         keys = [self.keys.ACCEL]
         if self.marionette.session_capabilities['platformName'] == 'LINUX':
-            keys.append(self.browser.get_localized_entity('searchFocusUnix.commandkey'))
+            keys.append(self.browser.get_localized_entity(
+                        'searchFocusUnix.commandkey'))
```

Comment:
Please also see the A-Team bootcamp: http://ateam-bootcamp.readthedocs.org/en/latest/reference/python-style.html. There is no exclusion for that at the moment. Just follow PEP8, and that would include 80 chars. Maybe that needs to be updated then too.